substrate/frame/assets/src/benchmarking.rs

```diff
@@ -553,5 +553,15 @@ benchmarks_instance_pallet! {
 		assert_last_event::<T, I>(Event::Blocked { asset_id: asset_id.into(), who: caller }.into());
 	}
 
+	transfer_all {
+		let amount = T::Balance::from(100u32);
```

Comment:
better use min_balance of the asset id here instead, multiply it by two for example.